testsuite/testsuite-microprofile-jwt/src/test/java/org/wildfly/swarm/microprofile/jwtauth/PrincipalLeakTest.java

```diff
@@ -57,14 +57,26 @@ protected static WARArchive initDeployment() {
     public void subjectShouldNotLeakToNonSecuredRequest() throws Exception {
         // project-no-roles-props.yml restricts the number of worker threads to 1,
         // that is, all requests are processed by the same single thread
-
```

Comment:
Same here, the `subjectShouldNotLeakToNonSecuredRequest` test will no longer verify what it used to verify.